LICENSE.md

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2015-2016
+Copyright (c) 2017
```

Comment:
how would you fix that? it depends on the project that you're using `ember init` on.